src/main/java/use_case/club_remove_member/ClubRemoveMemberInputData.java

```diff
@@ -4,16 +4,16 @@
  * Input data for the club remove member use case.
  */
```

Comment:
this was definitely the better choice, since usernames probably wouldn't be unique.

Reply:
I agree. We wanted something that would be unique and that didn't have anything to do with an ID number. Therefore, we settled on the email.